### ISSUE TITLE
Feature Request: Discord RPC Integration

### ISSUE THREAD

**Issue by Schlauer-Hax** [MultiMC/MultiMC5#3070]
I would like it if MultiMC shows your current Game, Version and/or Name of the Instance in Discord.

**Comment by sambhavsaggi** [comment]
Maybe something like this could help: https://github.com/MinnDevelopment/discord-rpc
Or this: https://github.com/Pizzabelly/EasyRP

**Comment by peterix** [comment]
Yet again, no. There is no point in doing this, as discord has no support for launchers.

**Comment by Blayung** [comment]
I'm sorry that I'm kicking into a closed issue, I didn't search well, so I made a duplicate of this issue here: #4831, but I wanted to suggest reopening this issue, because it's not true that discord isn't having support for launchers. I mean, it is supposed to only games be displayed in "Currently playing..." field in discord, but there's nothing bad about integrating the launcher of a game into it too. For example, Crystal launcher does it, and everyone is happy! :D  //+Sorry for my English, I wrote too much in strange way here so yea...
